extensions/widgets/navbar/navbar.lcb

```diff
@@ -391,6 +424,7 @@ public handler OnCreate() returns nothing
 
 	put tNavItems into mNavData
 
+	put "Android(Dark)" into mWidgetTheme
```

Comment:
iOS should probably still be the initial theme otherwise people will get a shock